nevergrad/parametrization/test_parameter.py

```diff
@@ -198,6 +198,16 @@ def test_instrumentation() -> None:
         inst = par.Instrumentation(scal, blublu=scal)
 
 
+def test_splitter() -> None:
+    inst = par.Instrumentation(par.Array(shape=(2,)), par.Array(shape=(3,)))
+    ranges = inst.get_ranges()
+    assert len(ranges) == 2
+    assert len(ranges[0]) == 2
+    assert len(ranges[1]) == 2
+    assert ranges[0][0] == 0 and ranges[0][1] == 2 and ranges[1][0] == 2 and ranges[1][1] == 5, f"We get{str(ranges)} instead of [[0,2],[2,5]]." 
```

Comment:
no need for `str(ranges)`, just `ranges`, the conversion is automatic
also, you can do assert 
expected = [(0,2),(2,5)]
assert ranges == expected
and then f"We get {ranges} instead of {expected}."